pymc_marketing/model_config.py

```diff
@@ -369,10 +383,66 @@ def handle_parameter_distributions(
     }
 
 
+def create_hierarchical_non_center(
+    name: str,
+    distribution_kwargs: dict[str, Any],
+    **kwargs,
+) -> pt.TensorVariable:
+    """
+    Create a hierarchical non-centered distribution.
+
+    This function constructs a hierarchical non-centered distribution using the provided
+    distribution parameters for offset, mu, and sigma. It returns a deterministic variable
+    representing the hierarchical non-centered distribution.
+
+    Parameters
+    ----------
+    name : str
+        The name of the variable.
+    distribution_kwargs : dict[str, Any]
+        A dictionary containing the distribution parameters for 'offset', 'mu', and 'sigma'.
+    **kwargs
+        Additional keyword arguments, including 'dims' for specifying desired dimensions.
+
+    Returns
+    -------
+    pt.TensorVariable
+        A PyMC deterministic variable representing the hierarchical non-centered distribution.
+
+    """
+    desired_dims = kwargs.get("dims", ())
+
+    mu_dist = distribution_kwargs["mu"]
+    mu_dims = mu_dist.get("dims", ())
+    sigma_dist = distribution_kwargs["sigma"]
+    sigma_dims = sigma_dist.get("dims", ())
+
+    offset = pm.Normal(name="offset", mu=0, sigma=2, dims=desired_dims)
+
+    mu_global = create_distribution(
+        f"{name}_mu_global",
+        mu_dist["dist"],
+        mu_dist["kwargs"],
+        dims=mu_dims,
+    )
+
+    sigma_global = create_distribution(
+        f"{name}_sigma_global",
+        sigma_dist["dist"],
+        sigma_dist["kwargs"],
+        dims=sigma_dims,
+    )
+
+    return pm.Deterministic(
+        name=name, var=mu_global + offset * sigma_global, dims=desired_dims
+    )
+
+
 def create_distribution(
     name: str,
     distribution_name: str,
     distribution_kwargs: dict[str, Any],
+    centered_hierarchy: bool | None = None,
```

Comment:
I prefer name centered because it is shorter. 

Just make bool = True as default

Reply:
I manage cases by default is `True` if exist. We must have default `None` otherwise when loop is `if not None` the values pass and mess up everything.

```python
centered_flag = parameter_config.get("centered", True)
```